src/frontend/src/pages/ProjectDetailPage/ProjectEdit/ProjectEditDetails.tsx

```diff
@@ -17,90 +17,112 @@ const ProjectEditDetails: React.FC<ProjectEditDetailsProps> = ({ users, control,
     <PageBlock title="Project Details">
       <Grid container spacing={1}>
         <Grid item xs={12} md={6} sx={{ mt: 2, mb: 1 }}>
-          <ReactHookTextField
-            name="name"
-            control={control}
-            sx={{ width: 10 / 10 }}
-            label="Project Name"
-            errorMessage={errors.name}
-          />
+          <FormControl sx={{ width: '60%' }}>
```

Comment:
if you set this to 80%, I think it should look fine, you can tweak that number to whatever you think looks best